doc/source/user/absolute-beginners-outline.rst

```diff
@@ -0,0 +1,137 @@
+**************
+NumPy: The Absolute Basics for Beginners
+**************
+
+This is a working outline for a future section introducing NumPy to absolute beginners. If you have comments or suggestions, please don’t hesitate to reach out!
+
+
+
+- How to install NumPy
+  
+  - various operating systems 
+
+- How to import NumPy
+
+- What is an array
+  
+  - What else might it be called
+  
+  - What is its shape
+
+  - Can you reshape it?
+
+  - What’s the difference between a Python List and a NumPy array? Why use NumPy?
+
+  - What are the attributes of an array?
+
+  - Broadcasting example
+
+- How to create an array (ndarray object)
+- How to create a basic array
+- How to create an array from existing data
+- How to create a new array from an existing array
+- How to specify the datatype
+  
+  - Examples of commonly used NumPy dtypes
+
+- How to inspect the size and shape of a NumPy array
+- How to check whether a list is empty or not
+- How to represent missing values and infinite values
+- Indexing and Slicing
+- Basic array operations (np.sum, np.dot, np.append, np.diff
+
+  - Operations on a single array
+
+  - Unary operators
+
+  - Binary operators
+
+  - Universal functions
+
+- How to compute mean, minimum, maximum, cumulative sum
+  
+  - (include row-wise and column-wise compute)
+
+- How to concatenate two arrays
+  
+  - column-wise
+
+  - row-wise
+
+    - np.concatenate, np.vstack, np.hstack
+
+- How to sort an array 
+  
+  - based on one (or more) columns
+    
+    - np.sort
+    
+    - np.argsort
+
+  - based on two or more columns
+    
+    - np.lexsort
+
+- How to pass a list of lists to create a matrix
+- How to extract specific items from an array
+- How to create sequences, repetitions, and random numbers
+
+  - np.linspace
+  
+  - np.logspace
+  
+  - np.zeros
+
+  - np.ones
+  
+  - np.tile
+
+  - np.random.randn
+  
+  - np.random.randint
+  
+  - np.random.random
+  
+  - np.random.choice
+  
+  - np.random.RandomState, np.random.seed
+
+- How to get the unique items and the counts
+- How to get index locations that satisfy a given condition 
+- How to reverse
+ 
+  - How to reverse the rows
+ 
+  - How to reverse the whole array
+
+- Reshaping and Flattening multidimensional arrays
+  
+  - flatten vs ravel
+
+- How to import and export data as a CSV
+- How to save and load NumPy objects
+- How to apply a function column-wise or row-wise
+- How to convert a 1D array into a 2D array (how to add a new axis)
+- Working with dates and times
```

Comment:
Hmm, "use Pandas" is probably the healthier advice in most real-world cases. Perhaps should be left out completely?